app/assets/stylesheets/v3/_settings.scss

```diff
@@ -211,8 +211,10 @@ $accordion-content-padding: 1rem;
 // 8. Accordion Menu
 // -----------------
 
-$accordionmenu-arrows: false;
-$accordionmenu-arrow-color: $primary-color;
+
+$accordionmenu-arrows: true;
+$accordionmenu-arrow-color: $white;
```

Comment:
we should also check this change on the other accordions, as well as the mobile navigation I _think_ this could possibly affect that as well.

Reply:
I'm okay with it being white, I ended up overriding it manually using the classes.

Reply:
This code shouldn't affect the other accordions, as this is the ["accordion menu" plugin](https://get.foundation/sites/docs/accordion-menu.html) , and the other accordions just use the ["accordion" plugin](https://get.foundation/sites/docs/accordion.html).